extensions/html/server/test/mocha.opts

```diff
@@ -1,3 +1,4 @@
 --ui tdd
 --useColors true
-./out/test
\ No newline at end of file
+./out/test
+./out/test/pathCompletion
```

Comment:
Should not be necessary when the file uses the right file extension: `pathCompletion.test.ts`